README.md

```diff
@@ -62,6 +62,18 @@ var mySubscribeForm = new SubscribeEmail({
 ### `url`
 **(Required for MailChimp)** A string of the `<form action="">` attribute generated by MailChimp which contains MailChimp authentication information. You can get this from MailChimp under "Signup forms > Embedded form code > Naked" and copying just the value from the `<form action="">` attribute. It should follow this format: `http://{username}.{data center}.list-manage.com/subscribe/post?u={user id}&id={list id}`.
 
+### `hiddenFields`
+An optional array of objects used to add `<input type="hidden">` elements to the default template to allow additional data to be submitted along with a form. As an example, on Universe forms this can be used to track an opt-in source using the [`lists_opt_in_source` parameter](https://github.com/SparkartGroupInc/sparkart-services/blob/master/doc/api/contacts.md);
```

Comment:
Unfortunately can't link to the Universe docs at the moment, that's a private repo. Maybe reference the MailChimp example instead?